projects/storefrontlib/src/layout/a11y/skip-link/service/skip-link.service.ts

```diff
@@ -47,17 +43,24 @@ export class SkipLinkService {
     }
   }
 
-  scrollToTarget(
-    target: HTMLElement,
-    position: SkipLinkScrollPosition,
-    event: MouseEvent
-  ): void {
-    target = <HTMLElement>target.parentNode;
-    (<HTMLElement>event.target).blur();
-    const options: ScrollIntoViewOptions =
-      position === SkipLinkScrollPosition.AFTER ? { inline: 'end' } : {};
+  scrollToTarget(skipLink: SkipLink): void {
+    const target =
+      skipLink.target instanceof HTMLElement
+        ? skipLink.target
+        : (skipLink.target as Element).parentElement;
 
-    target.scrollIntoView(options);
+    // we force a tabindex if not available, to ensure we can focus into the element
+    const currentTabIndex = target.getAttribute('tabindex');
```

Comment:
Should we check if the element is focusable? or is it assumed?

Reply:
The point of this, is that we make it a focusable element (for a moment), so that we can focus it. At line 62, we drop the index if it was added by us.